app/views/users/new.html.erb

```diff
@@ -0,0 +1,36 @@
+<!-- <form action="/users" method="post" accept-charset="UTF-8">
```

Comment:
It's preferable to use `users_path`

Reply:
We tried your suggestion like so:
`<form action="users_path" method="post" accept-charset="UTF-8">` with and without the quotes
but it doesn't work because this is the pure html form.